bundles/org.openhab.binding.knx/src/main/resources/OH-INF/config/channelConfig.xml

```diff
@@ -13,7 +13,7 @@
 		</parameter>
 		<parameter name="position" type="text" required="false">
 			<label>Address</label>
-			<description>The group address(es) in Group Address Notation to set the absolute position of the dimmer</description>
+			<description>The group address(es) in Group Address Notation to set the absolute brightness</description>
```

Comment:
I think the original description was more precise - firstly, because the connected device might be a heater or a speaker (yeah, I am e.g. using those for volume control) and secondly, because "absolute brightness" sounds as if it should be a value in lux or similar.

Reply:
removed from PR, porting smarthomej/addons#257 will override it anyway